abis_mapping/templates/survey_occurrence_data/templates/instructions.md

```diff
@@ -87,18 +85,18 @@ era](https://doi.org/10.1111/2041-210X.12594).
 
 <ins>Table 2: Suggested values for the controlled vocabulary fields in the template. Each term has
 a preferred label with a definition to aid understanding of its meaning. For some terms, alternative
-labels are provided that mean the same sort of thing. Note: <font color="red">geodeticDatum value
-must come from one of five options in this table.</font></ins>
+labels (with similar semantics) are provided. Note: <font color="red">geodeticDatum value
```

Comment:
a comma or two instead of the parentheses should be used here in place of the parentheses (or rewrite sentence).